packages/font-glyphs/src/space/index.ptl

```diff
@@ -14,7 +14,7 @@ glyph-block Spaces : begin
 		set currentGlyph.glyphRank   (9998)
 
 	create-glyph 'space' ' ' : glyph-proc
-		local df : include : DivFrame para.advanceScaleI
+		local df : include : DivFrame para.advanceScaleII
```

Comment:
I once did some experiments and found that `advanceScaleII` is too narrow. We could add a new advance scale gear specially for spaces.

Edit `parameters.toml` to set it.

Reply:
Done! I also made that param a valid metric override.